configs/projects/bolt-runtime.rb

```diff
@@ -125,6 +125,7 @@
   proj.component 'rubygem-minitar'
   proj.component 'rubygem-multi_json'
   proj.component 'rubygem-net-ssh'
+  proj.component 'rubygem-httpclient'
```

Comment:
Awesome! Assuming this means we can remove it from the bolt-vanagon repo?

Reply:
Yes, once this is merged and promoted. It's part of puppet-runtime builds for puppet-agent#master, so we can rely on it for bolt-server as well.